tests/graph_test.py

```diff
@@ -1,45 +1,22 @@
-from mythril.analysis.callgraph import generate_graph
 from mythril.mythril import MythrilAnalyzer, MythrilDisassembler
 from mythril.ethereum import util
 from mythril.solidity.soliditycontract import EVMContract
-from tests import (
-    BaseTestCase,
-    TESTDATA_INPUTS,
-    TESTDATA_OUTPUTS_EXPECTED,
-    TESTDATA_OUTPUTS_CURRENT,
-)
-import re
+from tests import TESTDATA_INPUTS
 
 
-class GraphTest(BaseTestCase):
-    def test_generate_graph(self):
-        for input_file in TESTDATA_INPUTS.iterdir():
-            output_expected = TESTDATA_OUTPUTS_EXPECTED / (
-                input_file.name + ".graph.html"
-            )
-            output_current = TESTDATA_OUTPUTS_CURRENT / (
-                input_file.name + ".graph.html"
-            )
+def test_generate_graph():
```

Comment:
Since you removed checking `input_file.graph.html`, do you want to you also remove the files themselves?

Reply:
Yeah, I planned on removing the directory after this get's merged. Or I can remove it now